tests/frontend/test_collections.py

```diff
@@ -1,3 +1,5 @@
+import time
```

Comment:
I don't think you used this

Reply:
Ugh, I forgot to clean up my imports. I'll remove it.

Reply:
We could add flake8 to help with that!